src/main/java/com/csforge/sstable/Query.java

```diff
@@ -0,0 +1,178 @@
+package com.csforge.sstable;
+
+import com.csforge.sstable.reader.CassandraReader;
+import com.csforge.sstable.reader.Partition;
+import org.apache.cassandra.config.*;
+import org.apache.cassandra.cql3.*;
+import org.apache.cassandra.cql3.restrictions.StatementRestrictions;
+import org.apache.cassandra.cql3.selection.Selection;
+import org.apache.cassandra.cql3.statements.*;
+import org.apache.cassandra.db.*;
+import org.apache.cassandra.db.filter.ClusteringIndexFilter;
+import org.apache.cassandra.db.filter.ClusteringIndexNamesFilter;
+import org.apache.cassandra.db.filter.ClusteringIndexSliceFilter;
+import org.apache.cassandra.db.filter.ColumnFilter;
+import org.apache.cassandra.db.partitions.UnfilteredPartitionIterator;
+import org.apache.cassandra.db.rows.UnfilteredRowIterator;
+import org.apache.cassandra.dht.Murmur3Partitioner;
+import org.apache.cassandra.exceptions.InvalidRequestException;
+import org.apache.cassandra.io.sstable.Descriptor;
+import org.apache.cassandra.io.sstable.format.SSTableReader;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import static org.mockito.Mockito.*;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.*;
+import java.util.stream.Stream;
+import java.util.stream.StreamSupport;
+
+public class Query {
+    private static final Logger logger = LoggerFactory.getLogger(Query.class);
+
+    public static QueryOptions OPTIONS = QueryOptions.DEFAULT;
+
+    static {
+        Config.setClientMode(true);
+
+        // Partitioner is not set in client mode.
+        if (DatabaseDescriptor.getPartitioner() == null)
```

Comment:
I don't think this is enough anymore to work completely so we should probably remove it.  Before we removed the option to provide the schema cql, this appeared to be enough, but now that we pull this information from the sstable and its accompanying files, we also need the validation data file among other things.